.travis.yml

```diff
@@ -14,5 +14,9 @@ script:
   - ./install.sh
   - docker-compose run --rm web createuser --superuser --email test@example.com --password test123TEST
   - docker-compose up -d
+  - docker container ls
```

Comment:
Please revert all the changes in this file as they seem for debugging purposes.

Reply:
I think they should stay, when the build is green you don't really need to look at the log. When there is a problem you want to see what is going on from the log without needing to do another commit. Since the logging commands do not add any significant time overhead  I would vote for leaving them where they are.

Reply:
I'm okay doing `docker-compose logs` in the `after_failure` handler in TravisCI but simply listing the containers with every run is not very useful. Also, we should be keeping the PR to the point, which is implementing Relay so please revert these changes.

Reply:
Created https://github.com/getsentry/onpremise/pull/443